packages/flutter_tools/lib/src/web/bootstrap.dart

```diff
@@ -472,3 +472,16 @@ String generateTestBootstrapFileContents(
 })();
 ''';
 }
+
+String generateDefaultFlutterBootstrapScript() {
+  return '''
+{{flutter_js}}
+{{flutter_build_config}}
```

Comment:
```suggestion
{{flutter_build_config}}
{{flutter_js}}
```

Reply:
Let's keep the default bootstrap as-is, we don't need to change the order.